xpano/gui/panels/bugreport_pane.cc

```diff
@@ -0,0 +1,78 @@
+#include "xpano/gui/panels/bugreport_pane.h"
+#include <iostream>
+
+#include <algorithm>
+#include <iterator>
+#include <sstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <imgui.h>
+
+#include "xpano/constants.h"
+#include "xpano/utils/future.h"
+#include "xpano/utils/text.h"
+#include "xpano/utils/sdl_.h"
+
+namespace xpano::gui {
+
+namespace {
+
+const std::string kGithubIssuesLinkText =
+    R"(https://github.com/krupkat/xpano/issues)";
+
+const std::string kTomasEmailText =
+    R"(tomas@krupkat.cz)";
+
+}  // namespace
+
+BugReportPane::BugReportPane(){}
+  
+void BugReportPane::Show() { show_ = true; }
+
+void BugReportPane::Draw() {
+  if (!show_) {
+    return;
+  }
+
+  ImGuiWindowFlags window_flags = ImGuiWindowFlags_NoDocking |
+                                  ImGuiWindowFlags_NoCollapse |
+                                  ImGuiWindowFlags_NoSavedSettings;
```

Comment:
Please test out with additional ImGuiWindowFlags_AlwaysAutoResize and removing the SetNextWindowSize call on the next line. The window should auto fit to its contents.

Docs:
https://github.com/ocornut/imgui/blob/588421986784df1ae3df16305d90cecdb07e9951/imgui.h#L959